UI.py

```diff
@@ -237,18 +237,32 @@ def CreateGetLeavesAcrossRangeOptions():
 
     return member_options
 
-def CreateMultipleLeavesEmbed(leaves_group, include_reason):
-    embed = discord.Embed(
-        title = f'Applied Leaves',
-        description = f'<@!{leaves_group[0][0]["member_id"]}> has requested:',
-        colour = 0x4682B4
-    )
-    embed.set_thumbnail(url = os.getenv("Leave_Balance_Link"))
-    embed.add_field(name = '\u200B', value = '\u200B', inline = False)
+def CreateMultipleLeavesEmbed(leaves, include_reason):
+    embeds_to_send = []
+    for leaves_group in leaves:
+        embed = discord.Embed(
+            title = f'Applied Leaves',
+            description = f'<@!{leaves_group[0][0]["member_id"]}> has requested:',
+            colour = 0x4682B4
+        )
+        embed.set_thumbnail(url = os.getenv("Leave_Balance_Link"))
```

Comment:
What does this link lead to? Or is this just for aesthetics?

Reply:
the link is for the photo that appears in the embed it self, so yeah it is just for aesthetics

Reply:
maybe `Leave_Balance_Image` is a better name